apps/prairielearn/elements/pl-excalidraw/pl-excalidraw.py

```diff
@@ -0,0 +1,172 @@
+import base64
+import json
+from enum import Enum
+from pathlib import Path
+
+import chevron
+import lxml.html
+import prairielearn as pl
+from lxml.html import HtmlElement
+from typing_extensions import assert_never
+
+
+class Attr(Enum):
```

Comment:
I think having this enum contributes to code bloat. You can just use the strings themselves as accessors. If we wind up doing an enum-based replacement for the getter methods later on, we'll want to make sure it can read type hints.

Reply:
I left the attribute strings as variables anyway because it makes refactoring hard. A bag of strings doesn't really play well with automated refactoring.